docs/log-viewer.qmd

```diff
@@ -161,14 +161,51 @@ The **Info** panel of the log viewer provides additional meta-information about
 
 ![](images/inspect-view-info.png){.border .lightbox fig-alt="The Info panel of the Inspect log viewer, displaying various details about the evaluation including dataset, plan, and scorer details, git revision, and model token usage."}
 
-## Viewer Embedding
+## Publishing
 
-The log viewer can be published to a static web server and linked to or embedded in another page (for example, in an iframe). To embed the viewer:
+::: {.callout-note appearance="simple"}
+Note that the `inspect view bundle` feature described below is available only in the development version of Inspect. You can install the development version with:
 
-1.  Copy the contents of the [`www/dist`](https://github.com/UKGovernmentBEIS/inspect_ai/tree/main/src/inspect_ai/_view/www/dist) directory (`src/inspect_ai/_view/www/dist`) to the folder you will be publishing.
+``` bash
+pip install git+https://github.com/ukgovernmentbeis/inspect_ai
+```
+:::
+
+You can use the command `inspect view bundle` (or the `inspect_view_bundle()` function from Python) to create a self contained directory with the log viewer and a set of logs for display. This directory can then be deployed as static HTML to any static web server (GitHub Pages, S3 buckets, etc...) to provide an interactive copy of the viewer. For example, to bundle the `logs` directory to a directory named `output`:
```

Comment:
Let's link to GitHub Pages explicitly as well as to the docs on S3 static hosting (https://docs.aws.amazon.com/AmazonS3/latest/userguide/WebsiteHosting.html). Maybe also link to Netlify?